_posts/2021-03-04-geoserver-2-19-rc-released.md

```diff
@@ -0,0 +1,195 @@
+
+  * Release candidates are a community building exercise and are not intended for production use.
+  * We ask the community (everyone: individuals, organizations, service providers) to download and thoroughly test this release candidate and report back.
+  * Participating in testing release candidates is a key expectation of our [open source social contract](http://www.ianturton.com/talks/foss4g.html#/). We make an effort to thank each person who tests in our release announcement and project presentations!
+  * GeoServer [commercial service providers](http://geoserver.org/support/) are fully expected to test on behalf of their customers.
+
+## Release Candidate Testing Priorities
+
+This is an exciting release and a lot of great new functionality has been added. We would like to ask for your assistance testing the following:
+
+  * The number one testing priority is to try out GeoServer with your data! _Mass market open source thrives on having many people to review. Scientific open source like GeoServer thrives on exposure to many datasets_.
+  * Help check that new [extension download bundles](https://sourceforge.net/projects/geoserver/files/GeoServer/2.19-RC/extensions/) have contain everything needed, including appropriate readme instructions and open source license information.
+  * The rest of this blog post highlights new features for GeoServer 2.19, please try out these features, read the documentation links, and ask questions.
+
+Known Issues:
+
+  * Layer configured with missing style throws NPE
+
+## MapML extension
+
+The MapML plugin has been under developer for some time as a community module and is now ready for a wider audience.
+
+(example)
+
+(screen snap)
+
+(documentation link)
+
+(thanks to contributor and customer for this development)
+
+## WPS JDBC extension
+
+The WPS JDBC extension allows to share the status of asynchronous WPS requests across a GeoServer cluster. The status of 
+all requests, past and ongoing, can be stored in a database, for later reference.
+
+The module uses GeoTools JDBC stores to access databases, create the necessary tables, and track status. 
+Connection parameters are provided as property files, e.g.:
+
+
+```
+user=postgres
+port=5432
+password=******
+passwd=******
+host=localhost
+database=gsstore
+driver=org.postgresql.Driver
+dbtype=postgis
+```
+
+For more information, refer to the [module documentation](https://docs.geoserver.org/latest/en/user/extensions/wps-jdbc/index.html).
+
+We'd like to thank Ian Turton for developing the module on behalf of GeoSolutions, Alessio Fabiani for providing
+documentation for it, and Andrea Aime for performing the QA and graduation steps.
+
+## WPS Download extension
+
+The WPS download plugin provides processes assisting in the download of large amounts of data,
+allowing to use asynchrounous request, when usage of WFS, WCS or WMS for the same job would lead to HTTP timeouts.
```

Comment:
The WPS download plugin assists in the download of large amounts of data, allowing asynchronous requests, where usaging of WFS, WCS or WMS for the same job would lead to HTTP timeouts.